thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/bao/jdbc/DataCompletenessConfigManagerImpl.java

```diff
@@ -0,0 +1,70 @@
+package com.linkedin.thirdeye.datalayer.bao.jdbc;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.linkedin.thirdeye.datalayer.bao.DataCompletenessConfigManager;
+import com.linkedin.thirdeye.datalayer.dto.DataCompletenessConfigDTO;
+import com.linkedin.thirdeye.datalayer.pojo.DataCompletenessConfigBean;
+import com.linkedin.thirdeye.datalayer.util.Predicate;
+
+public class DataCompletenessConfigManagerImpl extends AbstractManagerImpl<DataCompletenessConfigDTO> implements DataCompletenessConfigManager {
+
+
+  public DataCompletenessConfigManagerImpl() {
+    super(DataCompletenessConfigDTO.class, DataCompletenessConfigBean.class);
+  }
+
+  @Override
+  public List<DataCompletenessConfigDTO> findAllByDataset(String dataset) {
+    Predicate predicate = Predicate.EQ("dataset", dataset);
+    List<DataCompletenessConfigBean> list = genericPojoDao.get(predicate, DataCompletenessConfigBean.class);
+    return convertListOfBeanToDTO(list);
+  }
+
+  @Override
+  public List<DataCompletenessConfigDTO> findAllInTimeRange(long startTime, long endTime) {
+    Predicate timePredicate = Predicate.AND(Predicate.GT("dateToCheckInMS", startTime), Predicate.LT("dateToCheckInMS", endTime));
```

Comment:
Remember to come back to this line and decide whether start and end should be exclusive or inclusive, respectively.